### PR TITLE
chore: support publish package with ai key update

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

**Review by chagong — APPROVED** [review]

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -115,15 +115,42 @@ jobs:
         run: |
           npx lerna version --no-private --conventional-commits --conventional-prerelease --preid=rc --no-changelog --yes
 
+      - name: version stable npm packages to npmjs.org
+        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'stable' }}
+        run: |
+          npx lerna version --no-private --conventional-commits --conventional-graduate --no-changelog --yes
+
+      - name: update cli ai key
```
Comment (adashen) [inline .github/workflows/cd.yml:123]: the two steps share the same condition cannot be written in one step?

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -115,15 +115,42 @@ jobs:
...
         run: |
           npx lerna publish from-package --dist-tag=rc --yes
 
-      - name: release stable npm packages to npmjs.org
-        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'stable'}}
+      - name: publish stable npm packages to npmjs.org
+        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'stable' }}
         run: |
-          npx lerna publish --no-private --conventional-commits --conventional-graduate --no-changelog --yes
+          npx lerna publish from-package --yes
```
Comment (adashen) [inline .github/workflows/cd.yml:153]: by default the dist tag is latest right?

Comment (wenytang-ms) [reply .github/workflows/cd.yml:153]: yes

File: .github/workflows/cd.yml (hunk 1 above)
Comment (wenytang-ms) [reply .github/workflows/cd.yml:123]: this step use action can only modify one file once. or rewrite this action for two files' field change.

**Review by qinezh — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Pull request merged**